pyproject.toml

```diff
@@ -129,6 +129,8 @@ xlwt = "1.2.0"
 [tool.poetry.dev-dependencies]
 # flask only for moto[server]
 flask = ">=1.1.1"
+# Here and elsewhere we pin use of moto 1.3.7.
+# It would help to have a record of why. -kmp 27-Apr-2020
```

Comment:
The reason is after this version, a bug in moto is introduces involving SQS that breaks our code. We cannot upgrade versions until this bug is fixed, but moto has not had a "true" release in months, so we will have to stick with this version.

Reply:
Ah, OK. I can add this info in the file. Such info generally should always be there when pinning.

Reply:
This got fixed.